pkg/controller/servicemesh/controlplane/controller.go

```diff
@@ -272,12 +272,16 @@ func (r *ReconcileControlPlane) Reconcile(request reconcile.Request) (reconcile.
 }
 
 var reconcilers = map[string]*ControlPlaneReconciler{}
+var mu sync.Mutex
```

Comment:
We only create one `ReconcileControlPlane`, but shouldn't this mutex be contained within that struct?

Reply:
The mutex is to protect the `reconcilers` map, that's why it's declared here. But you're right. The map itself should be contained in the struct.

Reply:
Sorry I missed that.